server/routes/updateProfile.ts

```diff
@@ -77,7 +77,6 @@ const updateProfile = async (
   const addressesInSameChainbase = await models.Address.findAll({
     where: {
       user_id: address.user_id,
-      address: address.address,
```

Comment:
Can you explain to me why this was removed?

Reply:
Because for substrate, addresses in different chains are different based on ss58 prefix.